dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolProblem.java

```diff
@@ -89,7 +89,7 @@ public int hashCode() {
   public final String toString() {
     String jarInfo =
         containingClass != null
-            ? String.format("(%s) ", containingClass.getJar().getFileName())
+            ? String.format("(%s) ", containingClass.getJar().getJar().getFileName())
```

Comment:
more complex

Reply:
This toString  becomes ` String.format("(%s) ", containingClass.getJar())`, leveraging AnnotatedJar's toString which will print Maven coordinates if available.